srcs/incs/Location.hpp

```diff
@@ -1,11 +1,13 @@
 #ifndef SRCS_INCS_LOCATION_HPP_
 #define SRCS_INCS_LOCATION_HPP_
 #include <string>
+#include <vector>
 #include <CommonConfig.hpp>
 
 struct	Location {
-	CommonConfig	common;
-	/* More location settings */
+	CommonConfig			common;
```

Comment:
Still some common variables missing such as `root`, `err_page` etc?

Reply:
If I didn't miss anything, these are all the variables we need for now. We might need more if we decide to implement other functionalities.

Reply:
http://nginx.org/en/docs/http/ngx_http_core_module.html#location
What about the field named uri in this [syntax spec](http://nginx.org/en/docs/http/ngx_http_core_module.html#location):
```
syntax: | location [ = \| ~ \| ~* \| ^~ ] uri { ...}
```
I kind of need to name it, and it seemed to me that it was a good parameter for a possible location constructor. It doesn't appear to be optional. Do we not need/use it?

Thank you

Reply:
Well spotted @Damrod, we will need to store the path parameter :100: 
I think we also need to pass the CommonConfig of the server.

As per your recommendation, I updated the constructor:
```
Location(const std::string &path, const CommonConfig &common);
```
Does this look ok?

Reply:
Good, good.